dimmer.el

```diff
@@ -195,16 +195,7 @@ in ‘dimmer-face-color’."
     (dimmer-restore-all)))
 
 ;;;###autoload
-(defun dimmer-activate ()
-  "Activate the dimmer."
-  (interactive)
-  (dimmer-mode 1))
-
-;;;###autoload
-(defun dimmer-deactivate ()
-  "Deactivate the dimmer and restore all buffers to normal faces."
-  (interactive)
-  (dimmer-mode 0))
+(define-obsolete-function-alias 'dimmer-activate 'dimmer-mode)
```

Comment:
Should we `(make-obsolete dimmer-deactivate ...)` with a note to use `dimmer-mode` as well?  I don't know what's typical in this situation.

Reply:
I figured that at this early stage, nobody is likely to have written code that calls `dimmer-deactivate`, so it's not a big deal.